src/controllers/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('./../services/user.service');
+
+const getUsers = async (req, res) => {
+  try {
+    const users = await userService.getAllUsers();
+
+    res.send(users);
+  } catch (error) {
+    res.status(500).send('Server error');
+  }
+};
+
+const getOneUser = async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const user = await userService.getUserById(id);
+
+    if (!user) {
+      return res.status(404).send('User not found');
+    }
+    res.send(user);
+  } catch (error) {
+    res.status(500).send('Server error');
+  }
+};
+
+const createUser = async (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.sendStatus(400);
+  }
+
+  try {
+    const user = await userService.createUser(name);
+
+    res.status(201).send(user);
+  } catch (error) {
+    res.status(500).send('Server error');
+  }
+};
+
+const updateUser = async (req, res) => {
+  const { id } = req.params;
+  const { name } = req.body;
+
+  try {
+    const user = await userService.getUserById(id);
+
+    if (!user) {
+      return res.status(404).send('User not found');
+    }
+
+    if (typeof name !== 'string') {
+      return res.status(422).send('Invalid name');
```

Comment:
The check for 'Invalid name' is good, but consider adding a check for an empty string as well, to ensure the name is not just a string but a valid one.